package.json

```diff
@@ -36,6 +36,9 @@
 	},
 	"homepage": "https://github.com/stacysealky/donategifts#readme",
 	"dependencies": {
+		"@mantine/core": "^7.1.2",
+		"@mantine/dropzone": "^7.1.2",
+		"@mantine/hooks": "^7.1.2",
```

Comment:
please add them to the dev dependencies, we prebuild our react scripts and only ship the compiled stuff